tests/zfs-tests/tests/functional/online_offline/online_offline_002_neg.ksh

```diff
@@ -30,6 +30,7 @@
 #
 
 . $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/online_offline/online_offline.cfg
```

Comment:
If the cfg file is not used it does not work for physical disks -- we are doing all of our testing with physical disks so we need to use the cfg file or make a default that works with physical disks